app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<h4>TO DO:</h4>
+
+<% sorted = Task.sort.reverse %>
+
```

Comment:
On line 3, your view is talking directly to the model! This is a bit of an anti-pattern, and makes your code harder to read and reuse. Instead, you should put this line in your controller and save the result in an instance variable.